sdk/cosmosdb/cosmos/src/request/request.ts

```diff
@@ -127,6 +127,11 @@ export async function getHeaders({
     headers[Constants.HttpHeaders.ConsistencyLevel] = options.consistencyLevel;
   }
 
+  if (options.maxIntegratedCacheStalenessInMs && resourceType === ResourceType.item) {
+    headers[Constants.HttpHeaders.DedicatedGatewayPerRequestCacheStaleness] =
```

Comment:
Where is the validation that if the user passes value less than milliseconds, we will ignore it. 

Java SDK throws errors if the value is less than 1 millisecond. Please see this code for reference: 

```
public static long getMaxIntegratedCacheStalenessInMillis(DedicatedGatewayRequestOptions dedicatedGatewayRequestOptions) {
        Duration maxIntegratedCacheStaleness = dedicatedGatewayRequestOptions.getMaxIntegratedCacheStaleness();
        if (maxIntegratedCacheStaleness.toNanos() > 0 && maxIntegratedCacheStaleness.toMillis() <= 0) {
            throw new IllegalArgumentException("MaxIntegratedCacheStaleness granularity is milliseconds");
        }
        if (maxIntegratedCacheStaleness.toMillis() < 0) {
            throw new IllegalArgumentException("MaxIntegratedCacheStaleness duration cannot be negative");
        }
        return maxIntegratedCacheStaleness.toMillis();
    }
```